services/consensus_submit_message.proto

```diff
@@ -43,13 +60,25 @@ message ConsensusMessageChunkInfo {
     int32 total = 2;
 
     /**
-     * The sequence number (from 1 to total) of the current chunk in the message.
+     * The sequence number (from 1 to total) of the current chunk
+     * in the message.
      */
     int32 number = 3;
 }
 
 /**
- * UNDOCUMENTED
+ * Submit a message for consensus.
+ * <p>
```

Comment:
Should messages contain `<p>` tags or only blank lines 🤔

Reply:
Good catch, messages should just be blank lines, in order to remain clean _markdown_ syntax.
Occasional exceptions may be required _temporarily_ until we upgrade to a version of Java (23 or later) with the markdown syntax for Javadoc (JEP 467) available, and update PBJ to translate the documentation accordingly.

Reply:
Updated.